modules/apps/roles/roles-admin-web/src/main/resources/META-INF/resources/edit_role_permissions.jsp

```diff
@@ -65,7 +65,9 @@ if (!portletName.equals(PortletKeys.SERVER_ADMIN)) {
 	<liferay-util:include page="/edit_role_tabs.jsp" servletContext="<%= application %>" />
 </c:if>
 
-<aui:container cssClass="container-fluid container-fluid-max-xl container-form-lg" id="permissionContainer">
+<clay:container
```

Comment:
Why are we removing `container-form-lg` here? Is it intended?

Reply:
This is actually something I wanted to check today (this pr is not finished) I'm to going to look for the component in Portal and find out if we need just the padding https://github.com/liferay/clay/blob/0273c22d1f6a9dfbd22e39f9490d7a9da05e53b7/packages/clay-css/src/scss/variables/_globals.scss#L68 or also the menubar modification https://github.com/liferay/clay/blob/0273c22d1f6a9dfbd22e39f9490d7a9da05e53b7/packages/clay-css/src/scss/components/_grid.scss#L48

Reply:
Sounds good! 👍